### PR TITLE
chore: update cypress

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

**Comment by matthieu-foucault** [comment]
Cypress 7 is having significant performance issues (https://github.com/cypress-io/cypress/issues/15853), downgrading to 6

**Comment by matthieu-foucault** [comment]
Also https://github.com/cypress-io/cypress/issues/15779, so maybe 6.6 is the best option for now

**Comment by matthieu-foucault** [comment]
v6.9.1 seemed to be good after all. I had to fix some tests in `override-justification.spec.js` which should never have passed, but somehow did

**Review by pbastia — DISMISSED** [review]

File: app/cypress/support/commands.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ Cypress.Commands.add('login', (username, password) => {
 });
 
 Cypress.Commands.add('logout', () => {
-  cy.clearCookie('mocks.auth');
-  cy.request('/logout');
+  cy.get('header').contains('Logout').click();
+  cy.url().should('equal', Cypress.config().baseUrl + '/');
```
Comment (pbastia) [inline app/cypress/support/commands.js:38]: shouldn't we still clear the cookie?

Comment (matthieu-foucault) [reply app/cypress/support/commands.js:38]: no, `cy.logout` is only used when using the real login. Cypress will clear its own cookies automatically when using the mock login

**Review by pbastia — DISMISSED** [review]

**Review by pbastia — APPROVED** [review]

**Pull request merged**